pkg/orchestrator/connectors.go

```diff
@@ -163,3 +166,42 @@ func (c *ConnectorOrchestrator) Update(ctx context.Context, id string, config co
 	r.Skip()
 	return conn, nil
 }
+
+func (c *ConnectorOrchestrator) Validate(
```

Comment:
I think it would make sense to move the logic in this function into a function in the plugin service. The orchestration layer should ideally be as dumb as possible and just deal with transactions and orchestration, the logic right now is too aware of implementation details for my taste.

Or maybe we can have two functions in the plugin service (e.g. along the lines of `ValidateSourceConfig` and `ValidateDestinationConfig`), then the plugin service doesn't have to be aware of `connector.Type`.

Reply:
I tried adding this logic in the plugin service at first, but we would have to import the connector package, which would cause a circular dependency.
So I'll check the second approach